tools/libboard/board_n6000/board_n6000.c

```diff
@@ -400,7 +400,7 @@ fpga_result print_board_info(fpga_token token)
 		resval = res;
 	}
 
-	printf("FPGA SmartNIC N6010 Card \n");
+	printf("FPGA SmartNIC N6000 Card \n");
```

Comment:
Are you sure "FPGA SmartNIC" is correct?

Reply:
Change to Intel N6000 Acceleration Development Platform